js/web/lib/wasm/session-options.ts

```diff
@@ -64,34 +64,36 @@ const setExecutionProviders =
             epName = 'WEBNN';
             if (typeof ep !== 'string') {
               const webnnOptions = ep as InferenceSession.WebNNExecutionProviderOption;
-              if (webnnOptions?.deviceType) {
+              // const context = (webnnOptions as InferenceSession.WebNNOptionsWithMLContext)?.context;
```

Comment:
Is this a TODO?

Reply:
Yes. The current PR is only for API change. Another PR for implementation to support the new API is needed after this PR get merged.